### DISCUSSION TITLE
legendary: command not found

### DISCUSSION THREAD

**Discussion by DFauzan** [legendary-gl/legendary#361] [Q&A] [answered]
Excuse me, I'm a beginner at Linux. The problem is as entitled.

I installed legendary using:
`sudo apt install python3 python3-requests python3-setuptools-git
git clone https://github.com/derrod/legendary.git
cd legendary
pip install .`
in Konsole.

It's installed and I'm able to launch my game just fine. But, I have to input this:
`echo 'export PATH=$PATH:~/.local/bin' >> ~/.profile && source ~/.profile`
everytime before I play.

As far as I know, the code adds the line "export PATH=$PATH:~/.local/bin" to the ".profile" file. The line is already been added since the first input via Konsole, yet legendary wouldn't be detected unless I input that echo code, even though there are already 5 lines added to the ".profile" file since I keep inputting the code in Konsole before I play.
Any ideas as to how legendary would be detected without inputting that echo code? I'm using Ubuntu Studio 21.10.
Any help is appreciated! Any advises for my writing format is also appreciated!

**Comment by CommandMC** [comment, marked as answer]
Hi. This looks like the `.profile` file is not loaded by your shell.  
Could you send in the output of `echo $SHELL`?

**Reply by DFauzan** [reply]
Thank you for answering! The output:
/bin/bash

**Reply by CommandMC** [reply]
Huh, so you're using bash. Strange that it doesn't load `.profile` then...  
But we can work around that. Run `echo 'export PATH=$PATH:~/.local/bin' >> ~/.bashrc`, then close and re-open the Terminal. Now try to run `legendary` again

**Reply by DFauzan** [reply]
Hey, it works! My regards. I still don't understand why it wouldn't be detected before though. Thanks anyway!
